### ISSUE TITLE
[Feature request] Keyboard accessibility issues*

### ISSUE THREAD

**Issue by Ni9ja** [mengxiong10/vue-datepicker-next#43]
**What problem does this feature solve? : Keyboard accessibility issues**

Hello everyone! Could you please add "tabindex" for each date? For now all buttons are available, but, not a dates :(

**Comment by camya** [comment]
Yes. At the moment, it's not possible to use the Vue3 vue-datepicker-next using **keyboard only**. 

I tried vue2-datepicker, and the Vue2 version also does not support this. 

It whould be awesome and really helpful if you add this essential accessibility requirement.
